src/lib/permission-exclusions-keto-tuples.ts

```diff
@@ -28,40 +28,30 @@
  ******/
 
 import * as keto from '@ory/keto-client'
-import { PatchDelta, InternalRelationTuple } from '@ory/keto-client'
 import Config from '../shared/config'
 import { logger } from '../shared/logger'
+import { KETO_NAMESPACES, KETO_RELATIONS, PAGE_SIZE } from '../constants'
 
 class KetoTuples {
-  oryKetoReadApi: keto.ReadApi;
-  oryKetoWriteApi: keto.WriteApi;
+  relationshipApi: keto.RelationshipApi;
 
   constructor () {
-    this.oryKetoReadApi = new keto.ReadApi(
-      undefined,
-      Config.ORY_KETO_READ_SERVICE_URL
-    )
-    this.oryKetoWriteApi = new keto.WriteApi(
-      undefined,
-      Config.ORY_KETO_WRITE_SERVICE_URL
-    )
+    this.relationshipApi = new keto.RelationshipApi(undefined, Config.ORY_KETO_READ_SERVICE_URL)
```

Comment:
I think you need another object for write because write URL is different. Like here https://github.com/mojaloop/role-assignment-service/blob/15660f67a12224282b9788b51f70718bd284d06d/src/server/plugins/state.ts#L57

Reply:
Yep, you're right

Reply:
fixed